### PR TITLE
Update Images guidance and example of alt text

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#1887](https://github.com/alphagov/govuk-design-system/issues/1887).

This PR updates our [Images guidance's section about alt text](https://design-system.service.gov.uk/styles/images/#alternative-text) to specify:

- who our guidance is for
- redirect users who want guidance about publishing mainstream content on GOV.UK

We've suggested these changes because our Images guidance did not mirror the [recently updated GDS guidance on alt text](https://www.gov.uk/guidance/content-design/images#alt-text)

### CODE REVIEW

File: src/styles/images/index.md.njk
Comment (EoinShaughnessy) [inline src/styles/images/index.md.njk:38]: Much as I like cats, maybe we need a better example...

**Comment by joelanman** [comment]
I mentioned on the issue but copying here for visibility:

>I think we need to clarify our guidance, but just wondering if the content guidance for GOV.UK Publisher might be different to other services. For example if you use an icon in your service, it might help to add alt text. A different context to the content published on GOV.UK publishing.

So maybe something like "In most cases follow the gov.uk content thing, but if it wouldnt make sense to describe the image on the page, for example when using an icon, add alt text (example)"

I do follow that our current alt-text-in-body guidance means the alt text is available to all, which is a good thing. However the downside is that screen reader users are not aware of and cannot interact with images, for example to navigate the page, or download them.

**Comment by EoinShaughnessy** [comment]
If we update this guidance, should we also this PR to update any other images across the Design System site?

**Comment by EoinShaughnessy** [comment]
@calvin-lau-sig7 @joelanman Thanks for helping with this! Have tried to action your comments. Can I get another review? :)

Also - if we update this guidance, should we also use this PR to update any other images across the Design System site?

**Review by christopherthomasdesign — COMMENTED** [review]
Thanks for doing this, think it's great to add a lot more detail on when & how to use alt text.

I think the thing I got a bit confused by is how to include descriptive text within the body text, in a way that makes it clear you're referring to the image and that doesn't feel like a non-sequitur compared to the preceding content. Maybe an example that includes an image and some text would help – I'm not clear if we're talking about descriptive text within a paragraph or styled as a caption etc.

**Comment by EoinShaughnessy** [comment]
Blocked until we talk to more of the accessibility community.

**Comment by EoinShaughnessy** [comment]
@joelanman I've just added a commit that restores the original guidance.  Have also added 2 sentences (line 30) to say who the guidance is for, and to redirect users who need the 'publishers' guidance.

File: src/styles/images/index.md.njk
Comment (EoinShaughnessy) [inline src/styles/images/index.md.njk:52]:
```suggestion
Some images do not need descriptions. Use `alt=""` if the image is:
```

**Comment by EoinShaughnessy** [comment]
Hi @calvin-lau-sig7! This update has been around the houses, but after much investigation @joelanman and I thought it best to limit its scope. For now, we're just going to:

- tell users who the Design System guidance is for
- redirect them if they need guidance about publishing GOV.UK content.

[Full context in comment on original issue.](https://github.com/alphagov/govuk-design-system/issues/1887#issuecomment-979218049)

Could we get a review? :)

File: src/styles/images/index.md.njk
Comment (EoinShaughnessy) [inline src/styles/images/index.md.njk:31]: @calvin-lau-sig7 What do you reckon? :)

Comment (calvin-lau-sig7) [reply src/styles/images/index.md.njk:31]: Looks good! Could we be a bit more specific about what GOV.UK content is? 

Could say 'mainstream content' or perhaps say guidance, but we'd have to rewrite the sentence so we don't end up saying it twice to describe different things.

```suggestion
This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
```

Comment (EoinShaughnessy) [reply src/styles/images/index.md.njk:31]: @calvin-lau-sig7 That's a nice suggestion, thanks!

**Comment by EoinShaughnessy** [comment]
@calvin-lau-sig7 Thanks for reviewing! Have added in your suggestion. If all looks ok, can I get an approval?

**Review by calvin-lau-sig7 — APPROVED** [review]

**Comment by calvin-lau-sig7** [comment]
Certainly, done! I'll let you do the honours and merge it. 😄

**Comment by EoinShaughnessy** [comment]
@calvin-lau-sig7 Cheers for your help with this!

**Pull request merged**
